### ISSUE TITLE
FR: allow files to be changed without plan/apply

### ISSUE THREAD

**Issue by drevell** [abcxyz/guardian#324]
### TL;DR

As a codebase maintainer, I want to be able to make changes to a terraform directory that is already out of sync with production without triggering a terraform apply.

### Detailed design

Suppose we want to add a comment to every TF file in a repo (which has happened), or other codebase maintenance that is a no-op for the terraform resources created. Currently, we cannot touch a directory without trigger `plan` and `apply`.

Some ways this could work:

 - Kinda ugly, but the PR description could be parsed for special lines like `GUARDIAN_IGNORE=[my/dir,my/other/dir]`
 - Some kind of metadata/annotation on the PR? I don't think this exists though.

**Comment by sethvargo** [comment]
This already exists by proxy of GitHub Actions: https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs

**Comment by verbanicm** [comment]
> This already exists by proxy of GitHub Actions: https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs

This wont work because we use pull_request_target for Guardian see [here](https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs#:~:text=Note%3A%20Skip%20instructions%20only%20apply%20to%20the%20push%20and%20pull_request%20events.%20For%20example%2C%20adding%20%5Bskip%20ci%5D%20to%20a%20commit%20message%20won%27t%20stop%20a%20workflow%20that%27s%20triggered%20on%3A%20pull_request_target%20from%20running.).

**Comment by sethvargo** [comment]
We should probably adopt the same format and semantics then, with our own check.
